content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -462,40 +462,8 @@ To do this, install the OTLP exporter package:
 pip install opentelemetry-exporter-otlp
 ```
 
-Next, using the Flask server code from earlier, replace the console exporter
-with an OTLP exporter:
-
-```python
-from opentelemetry import trace
-from opentelemetry import metrics
-
-from random import randint
-from flask import Flask, request
-
-tracer = trace.get_tracer(__name__)
-meter = metrics.get_meter(__name__)
-
-roll_counter = meter.create_counter(
-    "roll_counter",
-    description="The number of rolls by roll value",
-)
-
-app = Flask(__name__)
-
-@app.route("/rolldice")
-def roll_dice():
-    return str(do_roll())
-
-def do_roll():
-    with tracer.start_as_current_span("do_roll") as rollspan:  
-        res = randint(1, 6)
-        rollspan.set_attribute("roll.value", res)
-        roll_counter.add(1, {"roll.value": res})
-        return res
-```
-
-By default, it will send telemetry to `localhost:4317`, which is what the collector
-is listening on.
+As exporters are logically separated from the trace acquiring, we do not 
+need to make any code changes.  
```

Comment:
```suggestion
The `opentelemetry-instrument` agent will detect the package you just installed
and default to OTLP export when it's run next.
```
Suggested rewording to this - the statement as-written wouldn't be true if we weren't using the agent, and that may confuse some people if they think it's a statement that would broadly apply.

Reply:
I am all for rewording, but as all this is on context of the agent, they would need to make more changes / be surprised anyway.
My text holds (somewhat) true anyway, that the code does not need to be changed when switching the exporter from console to OTLP. Of course as we don't manually instrument, we also don't need to configure the exporter, which would be required for manual mode.